### DISCUSSION TITLE
Improvement of NightCore audio at speeds below 1.5x

### DISCUSSION THREAD

**Discussion by vadrile** [ppy/osu#31074] [Q&A]
I'm not sure if this is the best way to describe it, but when using NightCore to lower the speed, the audio sounds like there are gaps between the sounds (similar to speaking in front of a fan).
This is especially noticeable at slower speeds, such as 1.05x.
As a NC enthusiast, I often switch to DT at lower speeds because of this issue.
I would greatly appreciate it if you could consider addressing it.
(I use machine translation.)

**Comment by peppy** [comment]
Just to confirm, this is the same thing you hear on stable right?

**Reply by vadrile** [reply]
I don't know how to reduce the speed of NC to below 1.5x in Stable.
At 1.5x, it sounds the same to me.
When the speed and pitch are synchronized (0.5x in replay), I can listen to it without any discomfort.

**Reply by vadrile** [reply]
Addendum:
Now that I think about it, it sounded exactly the same as during Half Time.
Sorry, I don't have much knowledge of music, so if this is something unavoidable, please don't worry about it...
As an alternative suggestion, I would be happy if a pitch adjustment option could be added to NC as well.

**Reply by KozacheShapka** [reply]
> As an alternative suggestion, I would be happy if a pitch adjustment option could be added to NC as well.

I support this suggestion! It would be great if this option was also available in DT-mod when you need to adjust pitch separately. This is very much missing.

**Comment by trustytrojan** [comment]
i think the expected behavior of nightcore is to have the pitch always be 1.5x the normal pitch, regardless of the playback rate. if you dont like the gaps, just use DT at 1.05x with adjust pitch enabled.

**Reply by vadrile** [reply]
The characteristics of NightCore songs are fast speed, high pitch (and kick drums), but just as these elements are not always at 1.5x, I don’t think the NC mod needs to have a 1.5x pitch to maintain its identity. In fact, if you enable pitch adjustment in DT and set it to around 1.2-1.3x, I think it can still be heard as a NightCore song without any noticeable discomfort.

If you lower it further, it feels off as a NightCore song, but that’s because the speed has been reduced too much.

I believe it’s fine to keep the default pitch at 1.5x while also offering pitch adjustment as an option.
(I’m using a translation tool, so I apologize if there are any misunderstandings.)
